Apps/Examples/Examples/All Examples/CustomAnchoredSymbolExample.swift

```diff
@@ -0,0 +1,321 @@
+                                sdf: false,
+                                stretchX: stretchX,
+                                stretchY: stretchY,
+                                content: imageContent)
+
+            try? style.addImage(image.tint(highlightTintColor),
+                                id: "imageRightHanded-Highlighted",
+                                sdf: false,
+                                stretchX: stretchX,
+                                stretchY: stretchY,
+                                content: imageContent)
+        }
+
+        // Left-hand pin
+        if let image =  UIImage(named: "ImageCalloutLeftHanded") {
+            try? style.addImage(image.tint(regularTintColor),
+                                id: "imageLeftHanded",
+                                sdf: false,
+                                stretchX: stretchX,
+                                stretchY: stretchY,
+                                content: imageContent)
+
+            try? style.addImage(image.tint(highlightTintColor),
+                                id: "imageLeftHanded-Highlighted",
+                                sdf: false,
+                                stretchX: stretchX,
+                                stretchY: stretchY,
+                                content: imageContent)
+        }
+    }
+
+    private func addFeatures() -> FeatureCollection {
+
+        var features = [Turf.Feature]()
+
+        let featureList = [
+            DebugFeature(coordinate: CLLocationCoordinate2DMake(40.714203, -74.006314), highlighted: false, sortOrder: 0, tailPosition: .left, label: "Chambers & Broadway - Lefthand Stem", imageName: "imageLeftHanded"),
+            DebugFeature(coordinate: CLLocationCoordinate2DMake(40.707918, -74.006008), highlighted: false, sortOrder: 0, tailPosition: .right, label: "Cliff & John - Righthand Stem", imageName: "imageRightHanded"),
+            DebugFeature(coordinate: CLLocationCoordinate2DMake(40.716281, -74.004526), highlighted: true, sortOrder: 1, tailPosition: .right, label: "Broadway & Worth - Right Highlighted", imageName: "imageRightHanded-Highlighted"),
+            DebugFeature(coordinate: CLLocationCoordinate2DMake(40.710194, -74.004248), highlighted: true, sortOrder: 1, tailPosition: .left, label: "Spruce & Gold - Left Highlighted", imageName: "imageLeftHanded-Highlighted"),
+            DebugFeature(coordinate: CLLocationCoordinate2DMake(40.7128, -74.0060), highlighted: true, sortOrder: 2, tailPosition: .center, label: "City Hall - Centered Highlighted", imageName: "imageCentered-Highlighted"),
+            DebugFeature(coordinate: CLLocationCoordinate2DMake(40.711427, -74.008614), highlighted: false, sortOrder: 3, tailPosition: .center, label: "Broadway & Vesey - Centered Stem", imageName: "imageCentered")
+        ]
+
+        for (index, feature) in featureList.enumerated() {
+            let point = Turf.Point(feature.coordinate)
```

Comment:
With https://github.com/mapbox/mapbox-maps-ios/pull/548/files#r673294649, in mind, maybe this would become something like…

```suggestion
        for (index, dataItem) in dataItems.enumerated() {
            let point = Turf.Point(dataItem.coordinate)
```